Ollivanders/src/net/pottercraft/Ollivanders2/Book/STANDARD_BOOK_OF_SPELLS_GRADE_1.java

```diff
@@ -28,6 +28,7 @@ public STANDARD_BOOK_OF_SPELLS_GRADE_1 (Ollivanders2 plugin)
       spells.add(O2SpellType.DIFFINDO);
       spells.add(O2SpellType.WINGARDIUM_LEVIOSA);
       spells.add(O2SpellType.INCENDIO);
+      spells.add(O2SpellType. PYROSVESTIRAS);
```

Comment:
Should have also added - is this really a first year spell? Looks like only Charlie Weasley ever does this in the books so seems like it is may be more advanced. Standard Book of Spells Grade 7 is pretty empty (mostly because no one knows what it has in it since Harry skipped his 7th year so it never appears in the books).